src/thirdparty/tests/CMakeLists.txt

```diff
@@ -2,19 +2,21 @@
 # other Axom Project Developers. See the top-level COPYRIGHT file for details.
 #
 # SPDX-License-Identifier: (BSD-3-Clause)
-
+#------------------------------------------------------------------------------
+# Collection of tests for axom's built-in third party libraries
```

Comment:
Nitpick: be consistent with "third party" vs "thirdparty"

Reply:
Done.  I went with "third party".